### PR TITLE
Seasonal Galore: The Squeakquel

### PR DESCRIPTION
**Pull request by Al-1ce**
## About The Pull Request

A refurbish of #12727 - mostly the same, minding the FAMAS and some grammar adjustments.
## Why It's Good For The Game

More variety is good. Nuff said.

## Changelog

:cl:
add: Adds more seasonal weapons
spellcheck: Fixed a few typos with seasonal listing
/:cl:

### CODE REVIEW

File: code/controllers/subsystem/persistence.dm [hunk 1]
```diff
@@ -223,6 +221,20 @@ SUBSYSTEM_DEF(persistence)
 		/obj/item/ammo_magazine/rifle/alf_machinecarbine = -1,
 		)
 
+/datum/season_datum/weapons/guns/rifle_seasonal_three
+	name = "Cold War"
+	description = "Hot Guns from the Cold War"
```
Comment (XSlayer300) [inline code/controllers/subsystem/persistence.dm:226]:
```suggestion
	description = "Hot guns from the Cold War"
```

File: code/controllers/subsystem/persistence.dm [hunk 2]
```diff
@@ -254,7 +266,7 @@ SUBSYSTEM_DEF(persistence)
 
 /datum/season_datum/weapons/guns/copsandrobbers_seasonal
 	name = "SWAT and terrorists"
```
Comment (XSlayer300) [inline code/controllers/subsystem/persistence.dm:268]:
```suggestion
	name = "SWAT and Terrorists"
```
CONSISTENCY

File: code/controllers/subsystem/persistence.dm [hunk 3]
```diff
@@ -275,3 +289,14 @@ SUBSYSTEM_DEF(persistence)
 		/obj/item/weapon/gun/shotgun/pump = -1,
 		/obj/item/weapon/gun/shotgun/pump/cmb = -1,
 		)
+
+/datum/season_datum/weapons/guns/lever_seasonal
+	name = "Lever actions"
```
Comment (XSlayer300) [inline code/controllers/subsystem/persistence.dm:294]:
```suggestion
	name = "Lever Actions"
```

**Comment by Al-1ce** [comment]
done

**Comment by Al-1ce** [comment]
done, but again.

**Review by Lumipharon — APPROVED** [review]

**Comment by FosterGrime-Heymynameslouie** [comment]
sepedmergh gaugahuh

**Pull request merged**